Assets/Scripts/UserInterface/UIButton.cs

```diff
@@ -0,0 +1,70 @@
+/**
+ * Author(s): Ranjini Das
+ **/
+
+using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+
+public class UIButton : MannBehavior
+{
```

Comment:
Just to be a bear about codebase conventions, I use Egyptian brackets. If you can forgive the particular-ness, I think we should keep this consistent across all classes.